From f5613e5e0ba98ac47aea0c99dc00b8f71c20e9b0 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 7 Feb 2022 14:59:54 +0100
Subject: [PATCH] pkglists/lxc: remove GnuPG things

Since LXC 4.0.12, developers decided to remove GPG and rely on HTTPS
only.

Reference:
https://github.com/lxc/lxc/commit/7c70b0d14396d20748084b30bac3d89da30ed8e2
---
 pkglists/lxc.lua.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkglists/lxc.lua.m4 b/pkglists/lxc.lua.m4
index b4623af..d89b188 100644
--- a/pkglists/lxc.lua.m4
+++ b/pkglists/lxc.lua.m4
@@ -7,6 +7,6 @@ forInstall(lxc,attach,auto,console,copy,create,destroy,freeze,info,ls,monitor,mo
 Install("luci-app-lxc", { priority = 40 })
 
 Install("kmod-veth", { priority = 40 })
-Install("gnupg", "gnupg-utils", "getopt", "tar", "wget", { priority = 40 })
+Install("getopt", "tar", "wget", { priority = 40 })
 
 _END_FEATURE_GUARD_
-- 
2.32.0

